components/scripts/lib/cli-parsers/gradle/05-cli-parser.m4

```diff
@@ -30,6 +31,7 @@ function print_help() {
   print_option_usage -p
   print_option_usage -t
   print_option_usage -a
+  print_option_usage "-y, --remote-build-cache-type" "Specifies the type of remote build cache to use in the second build run locally. Must be one of: 'http', 'gradle-enterprise', or 'develocity'"
```

Comment:
@etiennestuder I recently went through the scripts and removed all public facing usages of Gradle Enterprise. These changes now add some back. 

Something to consider is to not show `gradle-enterprise` in anything public facing, but still accept it as a value and still configure it. This is exactly what we did for the options like `--enable-gradle-enterprise`. 

On one hand it feels a bit unfortunate to regress on the "no public facing usages of GE", but on the other it feels bad implementing support for something that is immediately deprecated. 

We also need to determine how this functionality will work with 3.0. Will we still accept `gradle-enterprise` as a value?

Reply:
`develocity`, `gradle-enterprise`, `http`

Reply:
We decided to leave it. We can discuss what to do for 3.0 when the time comes.